showcase/src/app/demo/table/demo-table.component.ts

```diff
@@ -89,23 +95,30 @@ export class DemoTableComponent implements OnInit {
 				isFilterable: true
 			}
 		];
-
-		this.columns = [
+		this.columnsPropertiesCustomStyling = [
 			{
 				name: "id",
-				sortable: true
+				label: "id",
+				headerClassName: "bold",
+				isSortable: true,
+				isFilterable: true
 			},
 			{
 				name: "title",
-				sortable: true,
-				dataAccessor: (data: any): string => {
-					return "~" + data.title.label;
+				label: "Title",
+				cellFormatter: (value: any): string => {
+					return "~" + value.label;
 				},
-				compareFn: this.compareTitle
+				isSortable: true,
+				isFilterable: true,
+				compareFn: this.compareTitle,
+				cellClassNameFn: this.getTitleCellClassName
 			},
 			{
 				name: "description",
-				sortable: true
+				label: "Description",
+				isSortable: true,
+				isFilterable: true
```

Comment:
Can you also set a custom class on a row (`className` property) to demo this feature?